setup-2019-10-27.md

```diff
@@ -1,41 +1,227 @@
 # Setup guide for pilot libraries
 
-OCR-D's software is a modular collection of many projects (called _modules_) with many tools per
-project (called _processors_) that you can combine freely to achieve the workflow best suited for
-OCRing your content.
+OCR-D's software is a modular collection of many projects (called _modules_)
+with many tools per project (called _processors_) that you can combine freely
```

Comment:
per *module*?

Reply:
Combnine freely per module? Not sure what you mean here.

Reply:
I was referring to “per project”. In the line above you state that projects are called *modules*. So, I thought it would be useful to actually use this term in this line.

Reply:
Happy to reword this if it is not clear.

Reply:
oh right, yeah that makes sense.